core/server/models/base/bookshelf.js

```diff
@@ -78,7 +78,7 @@ ghostBookshelf.plugin('bookshelf-relations', {
                 };
 
                 // CASE: disable after hook for specific relations
-                if (['permissions_roles'].indexOf(existing.relatedData.joinTableName) !== -1) {
+                if (['permissions_roles', 'members_newsletters'].indexOf(existing.relatedData.joinTableName) !== -1) {
```

Comment:
We have a hook to update `sort_order` for pivot tables after update as base bookshelf plugin. Since new `members_newsletters` table doesn't has a `sort_order` column on it, this change disables the `after` hook from firing for that table as it will break the flow.